tests/fixtures/generate.php

```diff
@@ -132,7 +147,7 @@ function _file_info($filename) {
       'user_id' => [],
     ],
     '_attachments' => [
-      'files/0/effb530e-5529-46ed-bec4-1b8f05c274d6/public/file1.txt' => _file_info(FILE1),
+      'files:0:6f9e1f07-e713-4840-bf95-8326c8317800:public:file1.txt' => _file_info(FILE1),
```

Comment:
@dickolsson Yes, I've changed the delimiter t so that the Drupal file path doesn't get mixed up with the key.